openmdao/recorders/web_recorder.py

```diff
@@ -456,3 +406,9 @@ def convert_to_list(self, obj):
             return []
         else:
             return obj
+
+    def close(self):
+        """
+        Close.
+        """
+        pass
```

Comment:
It looks like `close` actually is necessary (for the SQLITE recorder it's what closes the connection to the DB). I'm going to add `close` back into the BaseRecorder and force subclasses to implement that method